commands/VariableCommand.js

```diff
@@ -272,6 +278,11 @@ VariableCommand.prototype = extend(BaseCommand.prototype, {
 		}
 		this.checkArguments(arguments);
 
+		if(variableName.length > this.maxNameLength) {
```

Comment:
The error will be "no matching device" since the first argument is supposed to be a device.

I think that message is OK. The command line help is quite clear: "specify a device id (or all) and a variable name." I would never have thought that if a device is not specified it will pick one (which one?)